rabbitmq/rabbitmq.go

```diff
@@ -0,0 +1,35 @@
+package rabbitmq
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+type RabbitMQClient struct {
+}
+
+func (r *RabbitMQClient) GetNumOfMessages(url, credentials string) (int, error) {
+	client := &http.Client{}
+	req, err := http.NewRequest("GET", url, nil)
+
+	if err != nil {
+		return -1, err
+	}
+	req.Header.Add("Authorization", fmt.Sprintf("Basic %s", credentials))
+
+	res, err := client.Do(req)
+	if err != nil {
+		return -1, err
+	}
+	defer res.Body.Close()
+
+	var result map[string]interface{}
```

Comment:
What about a struct with the desired fields? f.ex:
```go
type RQResponse struct {
    Messages float64 `json: messages`
}
```

Reply:
Make sense!

Doing 👀

Reply:
Done!

@drgarcia1986 can you check again pls? 🙏